src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chrome's http cache partitioning helps with better security and privacy.
+
+{# wf_published_on: 2020-10-01 #}
+{# wf_updated_on: 2020-10-01 #}
+{# wf_featured_image: /web/updates/images/generic/sd-card.png #}
+{# wf_tags: storage-isolation, cache #}
+{# wf_featured_snippet: Chrome's http cache partitioning helps with better security and privacy. #}
+{# wf_blink_components: Blink>Network #}
+
+# Gaining security and privacy by partitioning the cache {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+In general, caching can improve performance by storing data so future requests
+for the same data are served faster. For example, a cached resource from the
+network can avoid a round trip to the server. A cached computational result can
+omit the time to do the same calculation.
+
+In Chrome, the cache mechanism is used in various ways and HTTP Cache is one of
+such examples.
```

Comment:
```suggestion
In Chrome, the cache mechanism is used in various ways and HTTP Cache is one example.
```

Reply:
What you wrote was grammatically correct. It just wasn't how we would usually say it.